File: CodeWars/JavaScript/7Kyu/SimpleRemoveDuplicates.js
// URL: https://www.codewars.com/kata/simple-remove-duplicates/train/javascript

function solve(arr){
  let numbersFound = [];
  arr.reverse()

  console.log(arr);

  for (i=0;i<arr.length;i++){
    if (!numbersFound.includes(arr[i])){
      numbersFound.push(arr[i]);
    }
  }

  return numbersFound.reverse();
}

console.log(solve([3,4,4,3,6,3]));

File: CodeWars/Python/7Kyu/SimpleRemoveDuplicates.py
# URL: https://www.codewars.com/kata/simple-remove-duplicates/train/python

def solve(arr): 
  numbersFound = []

  arr = list(reversed(arr))

  for index, item in enumerate(arr):
    if (item not in numbersFound):
      numbersFound.append(item)
      
  return list(reversed(numbersFound))

print(solve([1,2,1,2,1,2,3]))

